docs/usage/type-translations.md

```diff
@@ -0,0 +1,252 @@
+(type-translations)=
```

Comment:
Could we for now keep this file as `type-conversions`, so that Github show the diff properly? Then later change the file name.

Reply:
Sure, will do that. I think the diff is pretty large either way though.

Reply:
Does github generally not understand `git mv`? It's funny because `git diff` gives a completely clear diff for these things.